### PR TITLE
[TI_Cybersixgill] Change from API to TAXII for the integration

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

The initial integration used to communicate with a third party python script, in collaboration with Cybersixgill we have converted this integration to rather use the TAXII server directly, this removes all third party tools requirements instead.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

The dashboards and fields are the same, but the ingest pipeline had to be rewritten to replace the functionality initially found in the python script

### CODE REVIEW

**Comment by P1llus** [comment]
Awaiting feedback from Cybersixgill on a replacement for the `Range` header, to retrieve more items than the current default limit of 100.
Everything else is ready to be reviewed though.

**Comment by P1llus** [comment]
I have now tested this with 90 days worth of data working successfully, it is fully ready for review

File: packages/ti_cybersixgill/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
-# Cybersixgill Webhook Integration
+# Cybersixgill Darkfeed TAXII Integration
 
-This integration creates an HTTP listener that accepts incoming HTTP requests from Cybersixgill integration script which retrieves indicators from [Cybersixgill Darkfeed](https://www.cybersixgill.com/products/darkfeed/).
+This integration collect with the commercial [Cybersixgill Darkfeed](https://www.cybersixgill.com/products/darkfeed/) TAXII server.
```
Comment (efd6) [inline packages/ti_cybersixgill/_dev/build/docs/README.md:3]: s/collect/connects/?

**Review by efd6 — COMMENTED** [review]
Need reformatting, check is complaining.

**Review by efd6 — APPROVED** [review]

**Comment by P1llus** [comment]
I have a demo scheduled sometime tomorrow and will proceed with merging after that

**Pull request merged**